src/tutorial/furthertopics/queues.rst

```diff
@@ -88,7 +97,13 @@ section like so:
                limit = 2  # Only 2 dessert dishes at one time.
                members = DESSERT
 
-Re-open the ``cylc gui`` if you have closed it and re-run the workflow.
+Install and play the workflow then open up the GUI (if you closed it)::
+
+   cylc validate .
+   cylc install --run-name tutorial-with-queues
+   cylc play queues-tutorial/with-queues
+   cylc gui
```

Comment:
Not all `cylc gui` code blocks include the "if you closed it" hint. Putting `cylc gui` into every list of commands is a bit OTT and could result in users opening multiple copies of the GUI.

Suggest introducing the GUI and how to launch it, then doing something like::

    ...
    # excluding the cylc gui command

```rst
Install and plat the workflow then :ref:`inspect it in the GUI`.
```

Also the tutorials could be done with Tui instead, which is especially useful as the GUI doesn't come with Cylc by default so may not be installed.

Perhaps we could say "view the workflow in a user interface" or something a bit more snappy.

Reply:
I'm happy to use TUI, it's just the red warning message at the top put me off using it. IMO it's more suitable for the sort of workflows we are running.